### PR TITLE
feat: add column for full name for masters students

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** Add column for full name

JIRA: [AU-218](https://2u-internal.atlassian.net/browse/AU-218)

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by jansenk** [comment]
Screenshot 2023-04-21 at 12 11 52 PM

File: src/components/GradesView/GradebookTable/Fields.jsx [hunk 1]
```diff
@@ -29,16 +29,16 @@ Username.propTypes = {
...
- * @param {string} email - email for display
+ * Fields.SimpleText
+ * Simple label field for text value.
+ * @param {string} value - value for display
  */
-const Email = ({ email }) => <span className="wrap-text-in-cell">{email}</span>;
-Email.propTypes = {
-  email: PropTypes.string.isRequired,
+const SimpleText = ({ value }) => (<span className="wrap-text-in-cell">{value}</span>);
+SimpleText.propTypes = {
```
Comment (nsprenkle) [inline src/components/GradesView/GradebookTable/Fields.jsx:37]: Nit: is there a reason to call it `SimpleText` and not just `Text`? Not a problem but is more verbose.

File: src/components/GradesView/GradebookTable/messages.js [hunk 2]
```diff
@@ -1,6 +1,11 @@
 import { defineMessages } from '@edx/frontend-platform/i18n';
 
 const messages = defineMessages({
+  fullNameHeading: {
+    id: 'gradebook.GradesView.table.headings.fullName',
+    defaultMessage: 'Full Name*',
```
Comment (nsprenkle) [inline src/components/GradesView/GradebookTable/messages.js:6]: I know this is a pattern we use other places but it is a bad pattern, putting the asterisk styling in the message itself. It should be added as a separate span or other styling.

```suggestion
    defaultMessage: 'Full Name',
```

**Review by nsprenkle — COMMENTED** [review]
While it wasn't technically specified as an AC, the ticket mentions an additional piece of work that is missing:
- default front-end disabled for large (ie, 100+ learners)

**Comment by jansenk** [comment]
> While it wasn't technically specified as an AC, the ticket mentions an additional piece of work that is missing:
> 
> * default front-end disabled for large (ie, 100+ learners)

Ryan's [most recent comment](https://2u-internal.atlassian.net/browse/AU-218?focusedCommentId=670432) on the ticket says
> hey friends. after a discussion with Mat I think we should replicate the functionality and visibility of the email field for >master-track users. 

>i.e. only show the name data for master-track seats 

This was a day after the text of the ticket was updated. I took this to mean that we were no longer doing the 100 student limit

**Comment by jansenk** [comment]
> I'll check with Ryan

Ryan confirmed that this is the desired behavior

**Review by nsprenkle — APPROVED** [review]
LGTM

**Pull request merged**